website/docs/opengl/c1/2-hello-quad.md

```diff
@@ -0,0 +1,524 @@
+* How to clear the window.
+* How the OpenGL pipeline works.
+* What vertex array objects, vertex buffers, and element array buffers are, and how to use them.
+* How to create and use shaders.
+* How to draw to the screen.
+
+This tutorial will feature a lot of content and new info, so we'll take it slow and with lots of explanations.
+
+## Initializing OpenGL
+The first thing we need to do before we can issue OpenGL commands, is to initialize it. Fortunately, Silk.NET windowing does most of the hard part for us, and we only have to get the GL api.
+
+Add the following using directive to the top of your file, you'll need this to use OpenGL:
+
+```cs
+using Silk.NET.OpenGL;
+```
+
+Then, at the top of your class, add the following:
+
+```cs
+private static GL _gl;
+```
+
+Finally, in your `OnLoad` method, add the following:
+
+```cs
+_gl = _window.CreateOpenGL();
+```
+
+What are we doing here? Silk.NET requires you to keep a **reference** to the OpenGL API. If you've used or seen OpenGL in C, you'll notice that this is different to the way that it is done there. This is done so that you can more easily keep track of multiple contexts. If you don't know what that is, don't worry about it for now, we won't be using it in these tutorials.
+
+<?# Info "If you take a look at the source in the Silk.NET samples repository, you will notice that it uses `_gl = GL.GetApi(_window)`. This is another way to get the GL API, however when using Silk.NET windowing, it's recommended that you use `_window.CreateOpenGL()` instead." /?>
+
+Now, run your application again. If all is good, you should see no change. Awesome! Let's do our first steps in OpenGL: Clearing the window.
+
+## Clearing the window
+Before we start this, let's take a look at what makes up an OpenGL window. 
+
+A window consists of several **framebuffers**. A framebuffer is essentially a texture that can be drawn to by OpenGL. An OpenGL window will consist of the following framebuffers:
+
+* Color buffer
+* Depth stencil buffer
+
+On top of this, a window will often contain at least two of these buffers. This is known as **double-buffering** and is imperitive for VSync to work properly. One buffer is displayed, while another is rendered to. 
+
+This is what makes up the **swapchain**, which, if you've ever looked into Direct3D, you should be familliar with. OpenGL does not use the term, nor does it allow you to manage this yourself, however it is still in the background, and it is helpful to know this.
+
+Now that we know this, we can get on with clearing the window.
+
+Add the following using directive to the top of your file:
+
+```cs
+using System.Drawing;
+```
+
+Then, in your `OnLoad` method, add the following:
+
+```cs
+_gl.ClearColor(Color.CornflowerBlue);
+```
+
+You may notice that this function contains various **overloads**, including ones for a Vector4, and 4 floats. We'll be using the overload that takes `System.Drawing.Color`, because it is the easiest to use and understand.
```

Comment:
```suggestion
You may notice that this function contains various **overloads**, including ones for a `System.Numerics.Vector4`, and 4 floats. We'll be using the overload that takes `System.Drawing.Color`, because it is the easiest to use and understand.
```